vscode/src/custom-prompts/menus/CustomCommandBuilderMenu.ts

```diff
@@ -10,46 +10,67 @@ const NewCustomCommandConfigMenuOptions = {
 }
 
 export interface CodyCommand {
-    title: string
+    slashCommand: string
     prompt: CodyPrompt
 }
 export class CustomCommandsBuilderMenu {
     public async start(commands: Map<string, CodyPrompt>): Promise<CodyCommand | null> {
+        // get slash command
+        const slashCommand = await this.makeSlashCommand(commands)
         // get name
-        const title = await this.makeName(commands)
+        const description = await this.makeDescription()
         // build prompt
-        const prompt = await this.makePrompt(title)
-
-        if (!title || !prompt) {
+        const prompt = await this.makePrompt()
+        if (!slashCommand || !description || !prompt) {
             return null
         }
-        void window.showInformationMessage(`New command: ${title} created successfully.`)
-        return { title, prompt }
+        void window.showInformationMessage(`New command: ${description} created successfully.`)
+        return { slashCommand, prompt: { ...prompt, description, slashCommand } }
     }
 
-    private async makeName(commands: Map<string, CodyPrompt>): Promise<string | undefined> {
-        const name = await window.showInputBox({
+    private async makeSlashCommand(commands: Map<string, CodyPrompt>): Promise<string | undefined> {
+        let slashCommand = await window.showInputBox({
             ...NewCustomCommandConfigMenuOptions,
-            prompt: 'Enter an unique name for the new command.',
-            placeHolder: 'e,g. Vulnerability Scanner',
+            prompt: 'Enter a keyword for this command to act as a slash command that you can run in chat or main quick pick.',
+            placeHolder: 'e.g. "explain" to assign /explain for the "Explain Code" command',
             ignoreFocusOut: true,
             validateInput: (input: string) => {
                 if (!input) {
-                    return 'Command name cannot be empty. Please enter a unique name.'
+                    return 'Slash command cannot be empty. Please enter a unique identifier.'
                 }
                 if (commands.has(input)) {
-                    return 'A command with the same name exists. Please enter a different name.'
+                    return 'A command with the same identifier exists. Please enter a different command name.'
```

Comment:
```suggestion
                    return 'A command with the same keyword exists. Please enter a different command name.'
```

we are calling it keyword above, should we keep the same word?

Reply:
Makes sense, addressed in https://github.com/sourcegraph/cody/pull/841/commits/66f5b3963b81b053acf095e17347cc24863b7613